src/components/pages/Map.tsx

```diff
@@ -54,6 +85,28 @@ function Map() {
   // },[]);
   // </座標取得 未実装>
 
+  // <表示範囲判定>
+  // const [selected, setSelected] = useState(null);
+  const [bounds, setBounds] = useState({ lat: 0, lng: 0 });
+
+  const mapRef = React.useRef<google.maps.Map | undefined>();
+
+  const onMapLoad = React.useCallback((map: google.maps.Map) => {
+    mapRef.current = map;
+  }, []);
+
+  const onMapBoundsChanged = React.useCallback(() => {
+    const latlngchange = mapRef.current.getBounds();
+    const lat: number = latlngchange.Ua.h;
+    const lng: number = latlngchange.Ua.j;
```

Comment:
undefined になりうるものには `?` をつけてオプショナルチェーンにしてみてください！
```suggestion
    const latlngchange = mapRef?.current?.getBounds();
    const lat: number = latlngchange?.Ua.h;
    const lng: number = latlngchange?.Ua.j;
```

そうすると `google.maps.LatLngBounds` 型には `Ua` というプロパティがないというエラーが出ると思いますが、ドキュメント見た感じ呼び出したいメソッドありましたかね？？
https://developers.google.com/maps/documentation/javascript/reference/coordinates#LatLngBounds